Tensile/TensileCreateLibrary.py

```diff
@@ -1109,6 +1109,47 @@ def param(key, value):
       
       param("best-solution", True)
 
+def generateLazyMasterFileList(masterFileList: List[Tuple[str, MasterSolutionLibrary]]) -> List[Tuple[str, MasterSolutionLibrary]]:
+    """ Generates a list of tuples that represent the name and the state associated with the lazy master libraries.
+
+    This function takes a list of MasterSolutionLibraries and traverses each lazy libraries.
+    It collects the items (i.e. the name and corresponding master file) and adds them to list
+    of master files.
+
+    Args:
+        masterLibraries: A list of name / master solution library pairs.
+    """
+    return masterFileList + list(itertools.chain.from_iterable((t for t in lib.lazyLibraries.items()) for _, lib in masterFileList))
```

Comment:
Since you're immediately casting to a list, I don't see the value of using a generator here, wouldn't this be simpler without the call to `itertools.chain.from_iterable` and equally performant?

Reply:
If we use list comprehension, we wind up with a list of lists and would still require a call to `itertools.chain.from_iterable`. I suspect the performance difference is negligible.

Reply:
Here's another option
```python
return [t for _, lib in masterFileList for t in lib.items()]
```